packages/presentation/lib/di/presentation_injector.dart

```diff
@@ -22,11 +25,21 @@ void _initSplashScreenModule() {
 }
 
 void _initHomeScreenModule() {
-  GetIt.I.registerFactory<HomeViewMapper>(() => HomeViewMapper());
+  GetIt.I.registerFactory<PresentationViewMapper>(
+    () => PresentationViewMapper(),
+  );
   GetIt.I.registerFactory<HomeBloc>(
     () => HomeBloc(
       GetIt.I.get<GetMovieListUseCase>(),
-      GetIt.I.get<HomeViewMapper>(),
+      GetIt.I.get<PresentationViewMapper>(),
```

Comment:
it's ok that each screen may have its own viewMapper

Reply:
unresolved. create plz separate viewmapper for each screen